structure/redblacktree/node.go

```diff
@@ -0,0 +1,23 @@
+package redblacktree
+
+// A representation of a tree node
+// Consists of Val (currently only int allowed)
+// left, right and parent are pointers to the relevant nodes
+// isRed is a representation of the color, made as bool
+type Node struct {
```

Comment:
Call it RBNode instead.
```suggestion
type RBNode struct {
```

Reply:
Completed changes